tests/end_to_end/models/participants.py

```diff
@@ -132,6 +132,8 @@ def modify_plan(self, new_rounds=None, num_collaborators=None, disable_client_au
             data = yaml.load(fp, Loader=yaml.FullLoader)
 
         data["aggregator"]["settings"]["rounds_to_train"] = int(self.rounds_to_train)
+        data["aggregator"]["settings"]["memleak_check"] = True
```

Comment:
in my opinion, we should enable this by default.

Reply:
Memory details are not needed for all runs. Basically it will be better for performance and Memory Leak related tests I think. 
Please let me know if still needs to change this.